waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -134,6 +135,15 @@ dbSeed genProjectDir seedName = do
     ExitSuccess -> Right ()
     ExitFailure c -> Left $ "Failed with exit code " <> show c
 
+dbIsRunning ::
+  Path' Abs (Dir ProjectRootDir) ->
+  IO Bool
+dbIsRunning genProjectDir = do
+  chan <- newChan
+  exitCode <- DbJobs.dbExecuteTest genProjectDir chan
```

Comment:
Seems like we don't really need a Job here, since we don't do anything with that Chan, right?
But I guess you used a Job because you had `runPrismaCommandAsDbJob`, which is very convenient? I guess it would be better if we had just normal `runPrismaCommand` or smth like that hm. But ok, we would still want to take over control over stdout possibly, hm. Ok, I guess if this works fine we can do it like this (although it feels like a tiny bit overkill, but I guess refactoring it would be even more of an overkill).